packages/react-jss/src/types/index.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import type {
+  InternalStyleSheetOptions,
+  Rule,
+  ToCssOptions,
+  RuleOptions,
+  JssStyle
+} from 'jss/lib/types'
+import RuleList from 'jss/lib/RuleList'
```

Comment:
should be imported from `jss` as a `type`

Reply:
Fixed